maze_transformer/evaluation/plot_maze.py

```diff
@@ -218,12 +227,11 @@ def plot(self, dpi: int = 100, title: str = "") -> MazePlot:
         self.ax.set_ylabel("row")
         self.fig.suptitle(title)
 
-        return self
-
     def show(self, dpi: int = 100, title: str = "") -> None:
-        """Plot the maze and paths and show the plot."""
+        """Plot the maze and paths and show the plot. DONT USE THIS IN NOTEBOOKS WHICH NEED TO BE TESTED IN CI!!!"""
```

Comment:
🤔 Can we display mazes in notebooks which are tested in CI some other way?

Reply:
I should add this to the comment, but in essence `MazePlot` has fig, ax objects and `MazePlot.plot()` will construct the figure. Then, just `pkt.plot()` plots it. Is this what you meant?